README.md

```diff
@@ -1,70 +1,159 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- Have [npm](https://www.npmjs.com/package/npm) installed in your project:
+  `curl -qL https://www.npmjs.com/install.sh | sh`
 
-### Making a Progressive Web App
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Clone this repository to your desired folder:
 
-### Advanced Configuration
+`git clone git remote add origin https://github.com/ClintonYounge/Mobile-Tech-Financial-Digest`
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+or
 
-### Deployment
+`git clone git remote add origin https://github.com/ClintonYounge/Mobile-Tech-Financial-Digest` <-- _Only if you have [SSH keys configured](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on your GitHub account_.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+or
 
-### `npm run build` fails to minify
+Utilizing Github Desktop. A simple interface, thats powerful enough to simplify this process.
+https://desktop.github.com/
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Install
+
+Run the `npm install` command in your console to install all necessary dependencies.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+Run the `npm run test` command in your console to run the tests.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Clinton Younge**
+
+- GitHub: [@ClintonYounge](https://github.com/ClintonYounge)
+- Linkedin: [@clinton-younge](https://www.linkedin.com/in/clinton-younge-83386a25a/)
+- Twitter: [@YoungeCjay](https://twitter.com/YoungeCjay)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Light mode / Dark mode / Color mode / Random gradient mode
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to leave a star to show your support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] It would be better if we give appropriate credit to the author of the design as required in the requirements. Here is a screenshot of the requirements
![Screenshot from 2023-06-27 23-03-02](https://github.com/ClintonYounge/Mobile-Tech-Financial-Digest/assets/46388113/9373f3da-3c3e-4817-928f-449b75178684)